lib/lexicon.py

```diff
@@ -49,11 +50,13 @@ def to_xml(self):
         for p in self.phon:
             el = ET.SubElement(res, "phon")
             el.text = p
-        for s in self.synt:
+        if self.synt and any(token.strip() for token in self.synt):
             el = ET.SubElement(res, "synt")
-            for t in s:
+            for token in self.synt:
+                if not token.strip():
+                    continue
                 el2 = ET.SubElement(el, "tok")
-                el2.text = t
+                el2.text = token
```

Comment:
not sure, but would you also use `token.strip()` here as you test for this beforehand?

Reply:
Yes makes sense! Will add that